tests/integration/integration_python_test.go

```diff
@@ -1519,6 +1519,19 @@ func TestPackageAddPython(t *testing.T) {
 				b, err := os.ReadFile(filepath.Join(e.CWD, "requirements.txt"))
 				assert.NoError(t, err)
 				assert.Contains(t, string(b), "sdks/random")
+
+				// Run the command again to ensure it doesn't add the dependency twice to requirements.txt
```

Comment:
Should we also check that the length of the file hasn't changed at all?

Reply:
Yeah that's a good idea